core/build.gradle

```diff
@@ -15,9 +15,9 @@ android {
 }
 
 dependencies {
-    implementation deps.appcompat
-    testImplementation deps.junit
-    testImplementation deps.mockito
+    implementation "androidx.appcompat:appcompat:${versions.appcompat}"
```

Comment:
Since I don't have too much experience with this does this means that when you use this version of the library your project also needs to use androidx.

Reply:
I don't think so. Implementation means that this is internal only dependency, it won't leak outside but it will be bundled with the library. Though I am not sure whether I am speaking the truth, will test it 👍

Reply:
you were right, I added README note with link to latest support.appcompat compatible version